akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowLimitWeightedSpec.scala

```diff
@@ -0,0 +1,63 @@
+package akka.stream.scaladsl
```

Comment:
I will also add the copyright for `StreamLimitReachedException.scala` in the PR for https://github.com/akka/akka/issues/19293 later